src/network/networkd-link.h

```diff
@@ -52,6 +52,7 @@ typedef struct Link {
         unsigned n_ref;
 
         int ifindex;
+        int master;
```

Comment:
The name `master` seems quite general. How about `netdev_master_ifindex` or something? Or, please add comments.